src/bitswap/bitswap_pb.proto

```diff
@@ -3,29 +3,42 @@ syntax = "proto3";
 package bitswap_pb;
 
 message Message {
+
   message Wantlist {
+    enum WantType {
+      Block = 0;
+      Have = 1;
+    }
+
     message Entry {
-      // the block cid (cidV0 in bitswap 1.0.0, cidV1 in bitswap 1.1.0)
-			bytes block = 1;
-      // the priority (normalized). default to 1
-			int32 priority = 2;
-      // whether this revokes an entry
-			bool cancel = 3;
+			bytes block = 1;		// the block cid (cidV0 in bitswap 1.0.0, cidV1 in bitswap 1.1.0)
```

Comment:
looks like mixed tabs and spaces?

Reply:
yes, it is a basically verbatim copy of the file from go-ipfs. Did not bother to format it. Does it matter?

Reply:
I'd have formatted it but it's done now